src/applications/simple-forms/40-0247/pages/additionalCertificatesRequest.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+  schema as addressSchema,
+  uiSchema as addressUiSchema,
+} from 'platform/forms/definitions/address.js';
+import definitions from 'vets-json-schema/dist/definitions.json';
+
+export default {
+  uiSchema: {
+    additionalAddress: addressUiSchema(
```

Comment:
Can you use the web component version here? addressUI() / addressSchema.  And for the title "Tell us where you'd like us..." can be a separate 'view:something'.

Reply:
Refactored to use v3 addressNoMilitaryUI & addressNoMilitarySchema.  Turns out we actually _can_ mix v3-web-components with older patterns/uiSchemas/schemas.